apps/idos-enclave/src/lib/enclave.js

```diff
@@ -48,10 +48,10 @@ export class Enclave {
     });
   }
 
-  storage(humanId, signerAddress, signerPublicKey, expectedUserEncryptionPublicKey) {
+  storage(humanId, signerAddress, signerEncryptionPublicKey, expectedUserEncryptionPublicKey) {
```

Comment:
This has to be wrong. Let's punt renaming this to https://github.com/idos-network/idos-sdk-js/pull/374